package.json

```diff
@@ -10,6 +10,8 @@
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
     "classnames": "^2.5.1",
+    "flowbite": "^2.3.0",
```

Comment:
used for the fastest lap standings. they have some other components we can use to save some time building ourselves